testsuite/tests/mgc/tlspolicy/test_cert_parameters.py

```diff
@@ -0,0 +1,67 @@
+"""Test parameters of TLS certificate generated by the TLSPolicy"""
+from datetime import datetime, timedelta
+
+import pytest
+from cryptography import x509
+
+pytestmark = [pytest.mark.mgc]
+
+
+@pytest.fixture(scope="module")
+def cert_params(module_label):
+    """Return tested certificate attributes"""
+    return {
+        "commonName": module_label,
+        "duration": "12h",  # <24h for Go time.ParseDuration and Python datetime.strptime compatibility
+        "usages": [
+            "digital signature",
+            "cert sign",
+            "crl sign",
+        ],
+        "privateKey": {
+            "algorithm": "ECDSA",
+            "size": 384,
+        },
+    }
+
+
+@pytest.fixture(scope="module")
+def dns_policy():
+    """Don't need DNSPolicy because only testing certificate generated by TLSPolicy"""
+    return None
+
+
+@pytest.fixture(scope="module")
+def tls_policy(tls_policy, cert_params):
+    """Update TLSPolicy with certificate parameters"""
+    tls_policy.model.spec.update(cert_params)
```

Comment:
I would prefer if you specified all the cert values here instead of just applying dict

Reply:
This version is slightly better, but I would still prefer updating the existing policy, than creating a new one

Reply:
One of the options how to achieve this, would be to make TLSPolicy dict-like and then use `tls_policy["commonName"] = commonName`

Reply:
good suggestion 👍